packages/web/src/components/NavBar/NavLink.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Link, useColorModeValue } from '@chakra-ui/react';
+
+export interface NavLinkProps {
+  label: string;
+  url: string;
+}
+export const NavLink: React.FC<NavLinkProps> = ({ label, url }) => (
+  <Link className="navbar" href={url}>
+    {label}
+  </Link>
+);
```

Comment:
In this case, since these will be navigating to other pages in the app, this is where you would want the `next/link` component wrapped around the Chakra link. The `passHref` option on the next link will tell it to not create an `a` tag and just wrap its child with the logic for smart routing. Also changed the `url` property to `href` to keep it consistent.

```suggestion
import NextLink from 'next/link';
import { Link, useColorModeValue } from '@chakra-ui/react';

export interface NavLinkProps {
  label: string;
  href: string;
}
export const NavLink: React.FC<NavLinkProps> = ({ label, href }) => (
  <NextLink href={href} passHref>
    <Link className="navbar">
      {label}
    </Link>
  </NextLink>
);
```